racket/src/cs/rumble/system.ss

```diff
@@ -122,6 +130,8 @@
 
 (define link-symbol
   (case (reflect-machine-type)
+    [(a6ios ta6ios arm64ios tarm64ios)
+     'framework]
     [(a6osx ta6osx i3osx ti3osx arm64osx tarm64osx)
      (if unix-style-macos?
          'static
```

Comment:
It would be good to check that `(system-type 'link)` as a whole is consistent with BC and with the docs in:

https://github.com/racket/racket/blob/244535d8b41169ee69d077cd327090c12f9dd969/pkgs/racket-doc/scribblings/reference/runtime.scrbl#L65-L72

The documentation for `system-type` doesn't mention the existence of `unix-style-macos?` builds at all, but the fact that it says `Mac OS` instead of `'macosx` could be interpreted to contradict the current possibility of returning `'static`. I also wonder if it would make sense to check `unix-link-shared?` there.

Reply:
I think it would probably be best to handle this in a separate PR.

Reply:
Yes, that's probably right. I'm not even entirely sure where or how `(system-type 'link)` is used.

Reply:
I've opened https://github.com/racket/racket/issues/5066 to keep track of this.